### PR TITLE
Report pinched cells

### PR DESCRIPTION
**Pull request by joakim-hove**
Problem with: https://github.com/OPM/opm-grid/pull/519 - only reported cells which were also joined with a pinch. Current version reports nnc pinching and removed cells independently.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/grid/CpGrid.hpp [hunk 1]
```diff
@@ -262,6 +262,12 @@ namespace Dune
         /// \param turn_normals if true, all normals will be turned. This is intended for handling inputs with wrong orientations.
         /// \param clip_z if true, the grid will be clipped so that the top and bottom will be planar.
         /// \param poreVolume pore volumes for use in MINPV processing, if asked for in deck
+        ///
+        /// The return value is a list of global indices to the cells which have
```
Comment (blattms) [inline opm/grid/CpGrid.hpp:266]: Any reason not to use `/// \return a list ,,,`

File: opm/grid/CpGrid.hpp [hunk 2]
```diff
@@ -262,6 +262,12 @@ namespace Dune
...
+        /// been removed in the grid processing due to small pore volume.
+        /// Observe that the processEclipseFormat() function only runs on rank
```
Comment (blattms) [inline opm/grid/CpGrid.hpp:268]: ... function only returns cell indices on rank 0. vector empty on other ranks.

**Review by blattms — COMMENTED** [review]
Small nitpicks

File: opm/grid/CpGrid.hpp (hunk 2 above)
Comment (joakim-hove) [reply opm/grid/CpGrid.hpp]: OK

File: opm/grid/CpGrid.hpp (hunk 1 above)
Comment (joakim-hove) [reply opm/grid/CpGrid.hpp]: No reason - apart from ignorance; updated now.

**Comment by blattms** [comment]
jenkins build this please

**Comment by bska** [comment]
> jenkins build this please

Just be aware that this is going to generate at least one regression failure.  The current reference solutions are out of date with respect to the master branch of opm-simulators.  I'm using OPM/opm-common#2424 to rectify that.

**Comment by bska** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Can this be merged?

**Comment by bska** [comment]
> Can this be merged?

If it's all the same I'd prefer to get #517 merged first.  I think it's **unlikely** that #517 will generate a merge conflict with this PR, but there's nevertheless a non-zero risk and this PR is smaller so conflict resolution should be easier here.

**Comment by joakim-hove** [comment]
> If it's all the same I'd prefer to get #517 merged first.

Well - no big deal; this PR is ready to be merged now - I don't know the readyness of #517?

**Comment by GitPaean** [comment]
I am okay this one goes in first.  #517 might takes time to review.  I will take care of it if something goes wrong.

**Comment by bska** [comment]
> I don't know the readiness of #517?

I was about to merge it, but then I [noticed](https://github.com/OPM/opm-grid/pull/517/files#r614862336) a change to the module requirements and now I think that it'll be at least until next week before that PR is ready.  I'll merge this now.

**Pull request merged**